analyses/chromosomal-instability/00-setup-breakpoint-data.R

```diff
@@ -356,7 +358,7 @@ breaks_density_list <- lapply(breaks_list, function(breaks_df) {
       samples, experimental_strategy, genome_size
     ) %>%
     # Count number of mutations for that sample
-    dplyr::summarize(breaks_count = dplyr::n()) %>%
+    dplyr::summarize(breaks_count = sum(!is.na(chrom))) %>%
```

Comment:
I just realized, while thinking about https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/490#issuecomment-585542405, that this may make some samples that should be NA into zeros. If the sample was not in the consensus seg file, it should be NA for CNV breaks and SV breaks  I *think* such a sample here would end up with a zero here.

Reply:
Those do end up with zeroes. If NA is preferred I can make those changes. I’ll just have to convert them back to zeroes for the CDF plots, unless we think those samples should be dropped from the CDF plots?

Reply:
Yes, drop them if they are missing. If you don't drop them, it says to your audience that there are _n_ samples that have 0 breakpoints when we don't have evidence either way.

Reply:
So is it correct to say that all WGS samples have been ran through both CNV detection pipelines, so if they don't show up in consensus CNV file or the SV file then they should be `NA`s for break density?

Reply:
If I'm understanding this correctly, then there will be no such thing as a `0`, only NAs where the minimum `break_counts` would 1.

Reply:
@cansavvy take a look at the documentation for https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/copy_number_consensus_call and also see https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/copy_number_consensus_call/results/uncalled_samples.tsv. I would also check the IDs in the  original files to be sure.

Reply:
The docs linked to should explain, but yes, there are both zeros and NAs. Some samples were deemed “uncallable” and should be NA. Others were called but had no CNVs and should be 0.